languages/csharp/examples/Program.cs

```diff
@@ -1,8 +1,8 @@
-﻿using Bit.Sdk;
+﻿using Bitwarden.Sdk;
 
 
-const string accessToken = "<access-token>";
-const string organizationIdStr = "<organization-id>";
+const string accessToken = "0.d37c384a-e70b-4239-bf2b-b06a0100efc8.1zxKrHPvHFzxaZ9XdI3UbUHu9jAvv2:MFWrA5ztEIJaKnKO2quJKw==";
```

Comment:
Hi @bbujak , you appear to have leaked your access token here, you may want to reset this even if it was just for testing :)

Reply:
Removed from code and revoked. And yes, it's just for testing. Thanks